js/bmlt_tabs_distance.js

```diff
@@ -0,0 +1,42 @@
+jQuery(document).ready(function($) {
+
+	if (navigator.geolocation) {
+		navigator.geolocation.getCurrentPosition(showLocation, errorHandler);
```

Comment:
won't this need an api key somewhere?

Reply:
no its kind of long to explain what the other function did... at one point if you hovered over the map link it would give you 3 options. 1. got to address in map view. 2. got to address in satellite view. 3. would give you link to get directions using your location as starting point. this was silly and too many options. if a user clicks on the google link to address using long/lats all you have to do is hit get directions and google does your location lookup.

anyway this uses this function
https://developer.mozilla.org/en-US/docs/Web/API/Geolocation/getCurrentPosition
tries to get your location and figures out distance from meeting long/lats

Reply:
https://github.com/radius314/crouton/blob/d7274e68d2539b2c959a568b0aa158d2411f3601/crouton.php#L805-L815

this is what i was talking about